lib/rules/comma-dangle.js

```diff
@@ -12,7 +12,9 @@
 //------------------------------------------------------------------------------
 
 module.exports = function (context) {
-    var forbidDanglingComma = context.options[0] !== "always";
+    var allowDangle = context.options[0] || "never";
```

Comment:
This is our convention: use a logical default. Rules aren't allow to throw.

There's a separate discussion around how rules should be validated here: #967